jbpm-task-assigning/kie-server-task-assigning-default-planner-kjar/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.kie.server</groupId>
+    <artifactId>jbpm-task-assigning</artifactId>
+    <version>7.34.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>kie-server-task-assigning-default-planner-kjar</artifactId>
```

Comment:
Seems like a dead code (isn't part of Maven build).

Reply:
Same here, this example project is for showing how we can extend the task assigning integration by providing a customized solver configuration, etc.
The final location probably optaplanner-examples-xxx will determined at some point.